lonabot/bot.py

```diff
@@ -176,8 +176,8 @@ async def init(self):
             schedreminder.SchedReminder(r.id, r.due)
             for r in self.db.iter_reminders()
         )
-        self._check_sched_task = self._loop.create_task(self._check_sched())
-        self._check_bday_task = self._loop.create_task(self._check_bday())
```

Comment:
Wait why was this still there?

Reply:
Yeah #32 does the exact same thing???

Reply:
Mysteries from ~~the universe~~ git. I have no idea.

Reply:
Nah @Sharparam's branch was a few commits behind master.